### PR TITLE
[DO NOT MERGE] feat(CX-3364): remove meet-the-specialists route and links

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Feat**

This PR solves [CX-3364]

### Description

Removed the `meet-the-specialists` route
NOT hidden behind a feature flag

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3364]: https://artsyproduct.atlassian.net/browse/CX-3364?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/Footer/Footer.tsx
Comment (dariakoko) [inline src/Components/Footer/Footer.tsx:142]: The changes are not hidden behind a feature flag. I believe this aligns with the requirements
In addition, removed the ff from here ^^^

**Review by MrSltun — APPROVED** [review]
Looks great!! 🌟 ✨ 
But I would advise that we don't merge these changes until we release the new design for the SWA landing page :)

**Comment by dariakoko** [comment]
Agree with you @MrSltun :)
I am not sure we can hide this type of changes behind a ff. Can we?

**Review by MounirDhahri — APPROVED** [review]
Nice

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
thanks for adding the redirect Daria

**Review by MrSltun — APPROVED** [review]
🚀

**Comment by damassi** [comment]
@dariakoko - If this is still needed, feel free to reopen PR and merge

**Pull request closed without merging**
